### ISSUE TITLE
Preview pane ceases to work after several changes

### ISSUE THREAD

**Issue by wongjn** [stackblitz/webcontainer-core#1111]
### Describe the bug

Using mainly Vite react project forks, after about 10–20 changes of the project, the preview pane stops working and shows a blank white screen, with the following in the browser console:

> Uncaught SyntaxError: Invalid or unexpected token (at vitejsvitex5jufe-5bg2--5173--a19575a2.local-credentialless.webcontainer.io/:1:39)

Looking in the *Sources* panel of devtools at the the error line highlights the following line:

```js
var 𝐰𝐜_0x525f = ['concat', 'catch', 'value', 'postMessage', 'css', 'filter', 'attributes', 'map', 'Unknown', 'sourceRoot', 'references', 'undefined', 'toString', 'body', '20038KwwdjZ', 'sourceMap', 'random', 'call', 'nonce', '__esModule', 'removeChild', 'close', '73qQQeMf', 'AAAA;EACE,YAAY;AACd;;AAEA;EACE,SAAS;EACT,UAAU;EACV,YAAY;AACd;;AAEA;EACE,cAAc;EACd,WAAW;EACX,YAAY;EACZ,SAAS;AACX', 'host', 'html\x20{\x0a\x20\x20height:\x20100%;\x0a}\x0a\x0abody\x20{\x0a\x20\x20margin:\x200;\x0a\x20\x20padding:\x200;\x0a\x20\x20height:\x20100%;\x0a}\x0a\x0aiframe\x20{\x0a\x20\x20display:\x20block;\x0a\x20\x20width:\x20100%;\x0a\x20\x20height:\x20100%;\x0a\x20\x20border:\x200;\x0a}\x0a', 'write', '1wuXwtc', 'test', 'msecs', 'then', 'Comlink.proxy', 'fetch', 'status', 'parentNode', 'line', 'stringify', 'set', 'slice', 'throw', 'singleton', 'preview.shared_worker.c604a0c4.js', 'reload', 'resolve', 'serviceWorker', '21vwePBu', 'start', '615957zPXrlq', 'constructor', 'keys', 'enqueue', 'function', '1679735wjkcnE', 'Connect', 'ping', 'exports', '631mOZAAJ', 'join', 'runtime.c604a0c4.js', 'msCrypto', 'splice', '52007eaHTOR', 'bind', 'error', 'substr', '133737evBbDN', 'toStringTag', 'all', '2477LXxCnS', 'appendChild', 'object', '16865Xllmam', 'nsecs', 'data', 'apply', 'defineProperty', 'media', 'length', 'default', 'Ready', 'message', 'addEventListener', '115syJEyq', 'register', '\x20*/', 'removeEventListener', 'type', 'createTextNode', 'string', 'Invalid\x20port\x20\x27', 'push', 'assign', 'version', 'Comlink.thrown', 'identifier', 'insert', '10gtcGJN', 'chunk', 'styleSheet'];
```

I believe it could be the `𝐰𝐜` characters at the start of the variable.

I can open the preview in a new tab and it works, but only for several more changes. By which time, similar erroneous behaviour is observed.

### Link to the blitz that caused the error

https://stackblitz.com/edit/vitejs-vite-6dimba?file=src%2FApp.jsx&terminal=dev

### Steps to reproduce

1. Open the project.
2. Let the `npm run dev` command start.
3. Make several changes until the error occurs. Preview pane ends up with white screen.

### Expected behavior

The preview pane should not ever be a white screen.

### Parity with Local

- [X] I have [run the project in my local machine](https://github.com/stackblitz/webcontainer-core/blob/main/repro.md#try-locally) and I could not reproduce the issue.

### Screenshots

White screen:
![image](https://github.com/stackblitz/webcontainer-core/assets/11310624/3957bd08-21a8-4f85-91f5-6b45bdba38cd)

The error reported in the browser console:
![image](https://github.com/stackblitz/webcontainer-core/assets/11310624/13d4e187-af17-455f-9fb7-03370918ffc5)

The erroneous code highlighted in *Sources* panel:
![image](https://github.com/stackblitz/webcontainer-core/assets/11310624/021c78fa-9ad0-41d0-8ef9-5bd1f89e22cd)

### Platform

```
Browser name  = Chrome
Full version  = 114.0.0.0
Major version = 114
navigator.appName = Netscape
navigator.userAgent = Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/114.0.0.0 Safari/537.36
performance.memory = {
  "totalJSHeapSize": 216036139,
  "usedJSHeapSize": 207522555,
  "jsHeapSizeLimit": 4294705152
}
Hash = a19575a2
```

**Comment by Nemikolh** [comment]
Hi @wongjn ! Thanks for the report and sorry for the annoyance! :pray: 

We had an encoding issue with that file at our CDN layer. I believe the issue has been fixed, is it still something you are experiencing?

**Comment by wongjn** [comment]
Wonderful news! I do not seem to be experiencing the bug anymore, thank you.

**Comment by wongjn** [comment]
Spoke too soon! Error is still occuring.

**Comment by BobbieGoede** [comment]
I'm experiencing the same issue but only when opening the preview in a new tab on Chrome on both Windows and macOS, this happens for any project (these work locally). For me the preview to the side works fine, but opening the preview in a new tab always results in a blank screen using Chrome.

<img width="501" alt="image" src="https://github.com/stackblitz/webcontainer-core/assets/6649305/56407bd8-e380-49aa-913d-8bc58dfea460">
<img width="929" alt="image" src="https://github.com/stackblitz/webcontainer-core/assets/6649305/7087cc7e-6bac-4afc-b8c3-46cb042fb831">

I tried it without extensions, and after signing out, after clearing site data and unregistering workers but no luck. 

I tried to replicate it (macOS) on several browsers (Safari, Firefox, Chrome Canary) but I am not running into the same issue in those browsers. 

Tested on several projects, last I tried was the Next.js starter https://stackblitz.com/edit/stackblitz-starters-qzwivp?description=The%20React%20framework%20for%20production&file=README.md&title=Next.js%20Starter
